test/skimNtuple2018.cpp

```diff
@@ -1949,251 +2221,349 @@ int main (int argc, char** argv)
-      if (lep2HasEES)
-      {
-        tlv_secondLepton_eleup.SetPxPyPzE (
-                         theBigTree.daughters_px_EleUp->at (secondDaughterIndex),
-                         theBigTree.daughters_py_EleUp->at (secondDaughterIndex),
-                         theBigTree.daughters_pz_EleUp->at (secondDaughterIndex),
-                         theBigTree.daughters_e_EleUp->at (secondDaughterIndex)
-                         );
-        tlv_secondLepton_eledown.SetPxPyPzE (
-                           theBigTree.daughters_px_TauDown->at (secondDaughterIndex),
-                           theBigTree.daughters_py_TauDown->at (secondDaughterIndex),
-                           theBigTree.daughters_pz_TauDown->at (secondDaughterIndex),
-                           theBigTree.daughters_e_TauDown->at (secondDaughterIndex)
-                           );
-      }
 
-      if (DEBUG)
+          theSmallTree.m_dau1_pt_tauup.push_back(tlv_firstLepton_tauup[idm].Pt());
+          theSmallTree.m_dau1_pt_taudown.push_back(tlv_firstLepton_taudown[idm].Pt());
+          theSmallTree.m_dau1_mass_tauup.push_back(tlv_firstLepton_tauup[idm].M());
+          theSmallTree.m_dau1_mass_taudown.push_back(tlv_firstLepton_taudown[idm].M());
+          theSmallTree.m_dau2_pt_tauup.push_back(tlv_secondLepton_tauup[idm].Pt());
+          theSmallTree.m_dau2_pt_taudown.push_back(tlv_secondLepton_taudown[idm].Pt());
+          theSmallTree.m_dau2_mass_tauup.push_back(tlv_secondLepton_tauup[idm].M());
+          theSmallTree.m_dau2_mass_taudown.push_back(tlv_secondLepton_taudown[idm].M());
+
+          if (idm < N_tauhDM_EES)
+            {
+              theSmallTree.m_dau1_pt_eleup.push_back(tlv_firstLepton_eleup[idm].Pt());
+              theSmallTree.m_dau1_pt_eledown.push_back(tlv_firstLepton_eledown[idm].Pt());
+              theSmallTree.m_dau1_mass_eleup.push_back(tlv_firstLepton_eleup[idm].M());
+              theSmallTree.m_dau1_mass_eledown.push_back(tlv_firstLepton_eledown[idm].M());
+              theSmallTree.m_dau2_pt_eleup.push_back(tlv_secondLepton_eleup[idm].Pt());
+              theSmallTree.m_dau2_pt_eledown.push_back(tlv_secondLepton_eledown[idm].Pt());
+              theSmallTree.m_dau2_mass_eleup.push_back(tlv_secondLepton_eleup[idm].M());
+              theSmallTree.m_dau2_mass_eledown.push_back(tlv_secondLepton_eledown[idm].M());
+            }
+        }
+
+      //MES:
+      double unc_MESup_first;
+      double unc_MESdw_first;
+      double unc_MESup_second;
+      double unc_MESdw_second;
+
+      unc_MESup_first = theBigTree.daughters_MESshiftup->at (firstDaughterIndex); // first daughter, DM 0
+      unc_MESdw_first = theBigTree.daughters_MESshiftdw->at (firstDaughterIndex); // first daughter, DM 0
+      unc_MESup_second = theBigTree.daughters_MESshiftup ->at (secondDaughterIndex); // second daughter, DM 0
+      unc_MESdw_second = theBigTree.daughters_MESshiftdw ->at (secondDaughterIndex); // second daughter, DM 0
```

Comment:
You probably just hinerited a mistake that I had made in skimNtuple2017, but can you fix the comments? 
// first daughter, DM 0 -> // first daughter, up
// first daughter, DM 0 -> // first daughter, dw
etc.